apps/web/src/services/asset-inventory/collector/collector-main/CollectorPage.vue

```diff
@@ -145,6 +148,8 @@ const initCollectorList = async () => {
     }
 };
 const refreshCollectorList = async () => {
+    state.initLoading = false;
+    state.hasCollectorList = true;
```

Comment:
question: Is it always `true` when refreshing?

Reply:
In this page, there are two separate sections where rendering occurs. To differentiate between them and ensure that the data is loaded only during the initial rendering, I set the initial value of the loading variable to true. When a `refreshCollectorList` function occurs, it is after the initial rendering, so I set hasCollectorList to false.